src/avfoundation.cs

```diff
@@ -3243,6 +3243,7 @@ interface AVCaptureSynchronizedMetadataObjectData
 
 	[NoMac, NoWatch, NoTV, iOS (11,0)]
 	[BaseType (typeof(AVCaptureSynchronizedData))]
+	[DisableDefaultCtor]
```

Comment:
That's a breaking change

Reply:
```
Type Changed: AVFoundation.AVCaptureSynchronizedDepthData

Removed constructor:

	public AVCaptureSynchronizedDepthData ();
```

Reply:
yes, it's logged in the xcode11 todo issue

Reply:
Ah yes I see, cool